### ISSUE TITLE
Conda 4.7.10 fails to solve environment while trying to install packages using conda install 

### ISSUE THREAD

**Issue by dupur** [conda/conda#9299]
I was trying to install pygame package using conda install command, which showed some conflicts. I aborted the installation using ctrl+c. After that conda started failing to solve environment everytime I try to use conda install. I removed anaconda 4.7.7 and installed 4.7.10 with no success. After trying a lot of things and couple of times of installation, `conda update conda -c conda-canary` shows there are some conflicts. I am a newbie, trying to solve this for last three days, any suggestions? I am on Linux Mint 18.03. 

Here are the outputs:

### `conda update conda -c conda-canary`
> Collecting package metadata (current_repodata.json): done
Solving environment: failed with current_repodata.json, will retry with next repodata source.
Initial quick solve with frozen env failed.  Unfreezing env and trying again.
Solving environment: failed with current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed
Initial quick solve with frozen env failed.  Unfreezing env and trying again.
Solving environment: failed
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:
anaconda==2019.07 -> python[version='2.7.16|3.6.8|3.7.3',build='h0371630_0|h9bab390_0|h0371630_0']
conda-env -> python[version='2.7.*|3.4.*|3.5.*']
If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to.  Your current python version
is (python=3.7).  Note that conda will not change your python version to a different minor version
unless you explicitly specify that.
The following specifications were found to be incompatible with a past
explicit spec that is not an explicit spec in this operation (conda-build):
conda -> conda-build[version='>=2.1|>=3']
conda -> conda-env[version='>=2.6']
The following specifications were found to be incompatible with each other:
Package enum34 conflicts for:
anaconda==2019.07 -> enum34==1.1.6=py27_1
conda -> enum34
conda-build -> enum34
conda[version='>=4.7.10'] -> enum34
Package futures conflicts for:
conda -> futures[version='>=3|>=3.0.0']
conda[version='>=4.7.10'] -> futures[version='>=3.0.0']
anaconda==2019.07 -> futures==3.3.0=py27_0
conda-build -> futures
Package psutil conflicts for:
anaconda-navigator -> psutil
anaconda==2019.07 -> psutil==5.6.3[build='py37h7b6447c_0|py36h7b6447c_0|py27h7b6447c_0']
conda-build -> psutil
Package statsmodels conflicts for:
anaconda==2019.07 -> statsmodels==0.10.0[build='py27hdd07704_0|py36hdd07704_0|py37hdd07704_0']
Package sphinxcontrib-serializinghtml conflicts for:
anaconda==2019.07 -> sphinxcontrib-serializinghtml==1.1.3=py_0
Package jupyterlab conflicts for:
anaconda==2019.07 -> jupyterlab[version='0.33.11|1.0.2|1.0.2',build='py36hf63ae98_0|py27_0|py37hf63ae98_0']
_ipyw_jlab_nb_ext_conf -> jupyterlab
Package scikit-learn conflicts for:
anaconda==2019.07 -> scikit-learn[version='0.20.3|0.21.2|0.21.2',build='py37hd81dba3_0|py36hd81dba3_0|py27hd81dba3_0']
Package libarchive conflicts for:
anaconda==2019.07 -> libarchive==3.3.3=h5d8350f_5
conda-build -> libarchive
..................
..................
<

### Conda Info:
> active environment : base
    active env location : /home/nahid/anaconda3
            shell level : 1
       user config file : /home/nahid/.condarc
 populated config files : /home/nahid/.condarc
          conda version : 4.7.10
    conda-build version : 3.18.8
         python version : 3.7.3.final.0
       virtual packages : __cuda=6.5
       base environment : /home/nahid/anaconda3  (writable)
           channel URLs : http://conda.anaconda.org/gurobi/linux-64
                          http://conda.anaconda.org/gurobi/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/nahid/anaconda3/pkgs
                          /home/nahid/.conda/pkgs
       envs directories : /home/nahid/anaconda3/envs
                          /home/nahid/.conda/envs
               platform : linux-64
             user-agent : conda/4.7.10 requests/2.22.0 CPython/3.7.3 Linux/4.15.0-64-generic linuxmint/18.3 glibc/2.23
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False

### Conda Source
>==> /home/nahid/.condarc <==
channel_priority: strict
channels:
   http://conda.anaconda.org/gurobi
  conda-forge
  defaults

>

**Comment by dupur** [comment]
Update: I created a new environment, installed the bare minimum packages I need there. Working fine for now. Still would be interested to see what I did wrong. :)

**Comment by soapy1** [comment]
It seems like you had some packages installed that are not available for python 3.7. Specifically this part of the message:
```
The following specifications were found
to be incompatible with the existing python installation in your environment:
anaconda==2019.07 -> python[version='2.7.16|3.6.8|3.7.3',build='h0371630_0|h9bab390_0|h0371630_0']
conda-env -> python[version='2.7.|3.4.|3.5.*']
If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Your current python version
is (python=3.7). Note that conda will not change your python version to a different minor version
```

indicates that conda-env is not available for your python 3.7 environment

**Comment by dupur** [comment]
> It seems like you had some packages installed that are not available for python 3.7. Specifically this part of the message:
> 
> ```
> The following specifications were found
> to be incompatible with the existing python installation in your environment:
> anaconda==2019.07 -> python[version='2.7.16|3.6.8|3.7.3',build='h0371630_0|h9bab390_0|h0371630_0']
> conda-env -> python[version='2.7.|3.4.|3.5.*']
> If python is on the left-most side of the chain, that's the version you've asked for.
> When python appears to the right, that indicates that the thing on the left is somehow
> not available for the python version you are constrained to. Your current python version
> is (python=3.7). Note that conda will not change your python version to a different minor version
> ```
> 
> indicates that conda-env is not available for your python 3.7 environment

If it makes any difference, I removed the previous conda using `rm -rf ~/anaconda3`. Then I downloaded the conda 4.7.10 and installed, the problem still persisted.
